### PR TITLE
Es6 syntax

### PR DESCRIPTION
**Pull request by lRebornsl**
# 🎏   In this *Milestone*  🙋‍♂️⁉:
- ✅ Follow the ES6 syntax.
- ✅ Implement Luxon library to show real date.
- ✅ Save the book's data to the local storage of browser.
- ✅ Retrieve data from local storage and fill them in related fields. 
- ✅ No linters error.
- ✅ Follow best github flow.

## ⭐To the code reviewer 👨‍💻 
💝 I really thank 🙇‍♂️  you, dear code reviewer 👨‍🎨  for dedicating your precious 🥇  time ⌚  to check the PR 🧾 .
- [x] I kindly ask 🙏 dear code reviewer 🤵 that if there is any kind of issue 🦯 in this project, please do list 📃 them in a descriptive 💡 manner and give your best suggestions 🎁 if needed.
- [x] If you think there is some big issues 👿 that is essential to be changed ♻️  please kindly contact 📞 me through *Zoom* or *Slack*, I'm available on Microverse UTC-6 time zone.

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (Deepakdanger) [inline modules/books.js]: - [ ] Kindly remove this & solve the linter manually.

File: modules/book-collection.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (Deepakdanger) [inline modules/book-collection.js]: - [ ] Kindly remove this & solve the linter manually.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @lRebornsl , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @lRebornsl , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**